.github/workflows/rust.yml

```diff
@@ -0,0 +1,22 @@
+name: Rust
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+env:
+  CARGO_TERM_COLOR: always
```

Comment:
There's no need to have color in the CI logs, furthermore, if someone wants to download them, I'm unsure if GitHub will ship them with the control characters